app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,337 @@
+            android:contentInsetLeft="20dp"
+            app:collapsedTitleGravity="start"
+            app:contentInsetLeft="20dp"
+            app:contentInsetStart="20dp"
+            app:contentScrim="@color/mainColor"
+            app:expandedTitleGravity="start"
+            app:expandedTitleTextAppearance="@color/mainColor"
+            app:layout_scrollFlags="scroll|exitUntilCollapsed"
+            tools:targetApi="lollipop">
+
+
+            <ImageView
+                android:id="@+id/movie_image_parallax"
+                android:layout_width="match_parent"
+                android:layout_height="306dp"
+                android:scaleType="centerCrop"
+                android:src="@drawable/movie_full"
+                app:layout_collapseMode="parallax"
+                app:layout_scrollFlags="scroll|enterAlways|enterAlwaysCollapsed"
+                android:contentDescription="@string/app_name" />
+
+
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+
+                <TextView
+                    android:id="@+id/movie_title"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="32dp"
+                    android:text="@string/movie_title"
+                    android:layout_above="@+id/movie_cat"
+                    android:textColor="@color/whiteColor"
+                    android:layout_marginBottom="10dp"
+                    android:textSize="22sp"
+                     />
+
+                <TextView
+                    android:id="@+id/movie_cat"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="32dp"
+                    android:text="@string/movie_category"
+                    android:layout_marginBottom="40dp"
+                    android:layout_alignParentBottom="true"
+                    android:textColor="@color/whiteColor"
+                    android:textSize="16sp"
+                  />
+
+
+            </RelativeLayout>
+
+            <androidx.appcompat.widget.Toolbar android:layout_width="match_parent"
+                android:layout_height="80dp"
+                android:elevation="2dp"
+                app:layout_collapseMode="pin">
+                <ImageView
+                    app:srcCompat="@drawable/ic__arrow_back"
+                    android:tint="@color/whiteColor"
+                    android:layout_width="@dimen/btnspacelayout_width"
+                    android:layout_height="@dimen/btnspacelayout_height"
+                    android:scaleType="centerInside"
+                    android:id="@+id/btnBack"
+                    android:textSize="22sp"
+                    android:layout_marginStart="20dp"
+                    android:contentDescription="@string/app_name" />
```

Comment:
Just a little note here. I don't think we need an image icon inside the toolbar.
Instead we should use `actionBar.setDisplayHomeAsUpEnabled(true)` inside activity / Fragment